autopush/tests/test_web_validation.py

```diff
@@ -865,6 +865,99 @@ def test_valid_vapid_crypto_header_webpush(self):
         eq_(errors, {})
         ok_("jwt" in result)
 
+    def test_valid_vapid_02_crypto_header_webpush(self):
+        schema = self._make_fut()
+
+        header = {"typ": "JWT", "alg": "ES256"}
+        payload = {"aud": "https://pusher_origin.example.com",
+                   "exp": int(time.time()) + 86400,
+                   "sub": "mailto:admin@example.com"}
+
+        token, crypto_key = self._gen_jwt(header, payload)
+        auth = "vapid t={token},k={key}".format(token=token,
```

Comment:
parse_endpoint collapses the whitespace of the header before splitting on =, so a valid case of "vapid k=..." will break